cocos/spine/lib/spine-version.ts

```diff
@@ -23,3 +23,11 @@
 */
 
 export const SPINE_VERSION = '3.8';
+
+export function isBinarySupported (buffer: Uint8Array): boolean {
+    return false;
+}
+
+export function isJsonSupported (json: JSON): boolean {
```

Comment:
Add some comments for these two methods since they're only avaiable in editor.

And how about rename the method to `isBinaryCompatible` | `isJsonCompatible` ?